.github/ISSUE_TEMPLATE.md

```diff
@@ -2,8 +2,8 @@ Hi there,
 
 Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.
 
-### Terraform Version
-Run `terraform -v` to show the version. If you are not running the latest version of Terraform, please upgrade because your issue may have already been fixed.
+### Terraform and Akamai Terraform Provider Versions
+Run `terraform -v` in the directory with the .tf files to show the terraform and /akamai provider versions. If you are not running the latest versions of Terraform and Akamai Terraform Provider, please upgrade because your issue may have already been fixed.
```

Comment:
That reminded me that we need to add support for latest Terraform version as we didn't bump that for quite some time. I'll find a ticket for it